src/mds/MDSMap.cc

```diff
@@ -863,8 +871,24 @@ void MDSMap::decode(bufferlist::const_iterator& p)
   } else if (ev > 14) {
     decode(min_compat_client, p);
   }
-
+  if (decode_ev)
+    decode(encode_ev, p);
   DECODE_FINISH(p);
+
+  // ev part
+  if (decode_ev && !encode_ev) {
```

Comment:
this logic is reverse

Reply:
@ukernel Fixed. Happened because of find/replace of the previous variable name `is_client`.